LICENSE-3rdparty.csv

```diff
@@ -4,7 +4,7 @@ Datadog.Trace,https://github.com/damianh/LibLog,MIT,Copyright (C) 2011-2017 Dami
 Datadog.Trace,https://github.com/serilog/serilog,Apache-2.0,Copyright (c) 2013-2018 Serilog Contributors
 Datadog.Trace,https://github.com/serilog/serilog-sinks-file,Apache-2.0,Copyright (c) 2016 Serilog Contributors
 Datadog.Trace,https://github.com/JamesNK/Newtonsoft.Json,MIT,Copyright (c) 2007 James Newton-King
-Datadog.Trace.ClrProfiler.Managed,https://github.com/kevin-montrose/Sigil,MS-PL,2013-2016 Kevin Montrose
+Datadog.Trace,https://github.com/kevin-montrose/Sigil,MS-PL,2013-2016 Kevin Montrose
```

Comment:
We don't use Sigil anymore so we can actually remove this

Reply:
Thanks. Removed in a3780dd5877d74e3cd3550bdfc741dc6ab543806.